### DISCUSSION TITLE
Weekly Sync - 12th Sept 2022

### DISCUSSION THREAD

**Discussion by matdehaast** [interledger/rafiki#597] [General]
## Updates

Brandon
 * [be able to query open payments payment using magic token](https://github.com/interledger/rafiki/pull/595)
 * add support to rafiki for open payments connections

Sabine
 * [allow non-interactive grants for incoming payments](https://github.com/interledger/rafiki/pull/588)
 * working on adding clientId to OP resources to be able to query based on client created resources

Raphael
 * [waiting on PR review for TB helm chart](https://github.com/interledger/rafiki/pull/403/files)
 * working on helm chart for actual rafiki components

Alex
 * Postman collection has some minor updates
 * Trying to ensure that the postman collection stays in sync with the repo
 * Andrei joins from breakpoint to work on Mojaloop/Rafiki integration

Damien
 * working on mock iDP that test the AS.

DJ
 * [compaction being refactored](https://github.com/tigerbeetledb/tigerbeetle/pull/146) that should hopefully resolve most of the assertion bugs

Adrian
 * working on doing a port of the auth package to adonisJs.
 * required some extra work to use libraries for parsing and validation
 * omer helping out with the implementation.

**Comment by sabineschaller** [comment]
[Rafiki Summit Closing Overview](https://coil.notion.site/2022-Rafiki-Summit-Closing-Overview-6a2e4b271af9469690f5040bff79e6f9)
